attribution-data-file-share/src/test/java/gov/cms/ab2d/attributionDataShare/AttributionDataShareTest.java

```diff
@@ -0,0 +1,35 @@
+package gov.cms.ab2d.attributionDataShare;
+
+import gov.cms.ab2d.attributionDataShare.AttributionDataShare;
+import gov.cms.ab2d.testutils.AB2DPostgresqlContainer;
+import gov.cms.ab2d.testutils.TestContext;
+
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+@Testcontainers
+public class AttributionDataShareTest {
+
+    @SuppressWarnings({"rawtypes", "unused"})
+    @Container
+    private static final PostgreSQLContainer POSTGRE_SQL_CONTAINER = new AB2DPostgresqlContainer();
+
+    @Test
+    void attributionDataShareInvoke() {
+        AttributionDataShare attributionDataShare = new AttributionDataShare();
+
+        assertDoesNotThrow(() -> {
+            attributionDataShare.handleRequest(new TestContext());
+        });
+
+        String filePath = attributionDataShare.handleRequest(new TestContext());
```

Comment:
mmm.. you just executed the handler and run it again just to get a file name.
Now I see why you want to return it from handler)

Reply:
In hindsight, the filePath variable should have been initialized above the assert statement to null or empty string, like:

`String filePath = "";`

Then in the assert I could have done:

`String filePath = attributionDataShare.handleRequest(new TestContext());`

And then lastly

'System.out.println("AttributionDataShareTest: The file path is: " + filePath);'

Or whatever else we need to do to verify the test (the println isn't needed but would have been used for testing locally). Ideally we would read the file here and verify the format and potentially the contents for validity?